ocs_ci/ocs/node.py

```diff
@@ -56,8 +56,8 @@ def get_typed_nodes(node_type='worker', num_of_nodes=None):
     """
     nodes = get_node_objs()
```

Comment:
I believe we can skip the assignment and use get_nodes_objs()` inline in the list comprehension below

Reply:
fixed